pygrocy/utils.py

```diff
@@ -5,7 +5,7 @@
 
 
 def parse_date(input_value):
-    if input_value is None:
+    if input_value is "" or input_value is None:
```

Comment:
This seems like to be easy to test. Would you add a test in https://github.com/SebRut/pygrocy/blob/develop/test/test_utils.py please? :)

Reply:
Of course! I'm just not used to the testing :D